### PR TITLE
CC88982,88983: Rephrasing sentences according to contributor suggestion.

### PR DESCRIPTION
**Pull request by v-maudel**
Hello, @mairaw  ,
This proposed file change comes from https://github.com/dotnet/dotnet-api-docs.ja-jp/pull/15.
Could you review this contribution and help to merge if agreed?
Many thanks in advance.

### CODE REVIEW

File: xml/System.Windows/UIElement.xml [hunk 1]
```diff
@@ -1775,10 +1775,13 @@ Image with an Elliptical Clip Region
 ## Remarks  
  To be focusable, <xref:System.Windows.UIElement.Focusable%2A> and <xref:System.Windows.UIElement.IsEnabled%2A> must both be `true`.  
   
- Even if an element is focusable and enabled, event handling within a specific tree, (such as for a composite control) might respond to the preview focus events by not allowing focus there, thus this method would return `false`.  
-  
- Focus in general is governed by two separate concepts: keyboard focus and logical focus, which are not always identical.  For more information, see [Focus Overview](~/docs/framework/wpf/advanced/focus-overview.md) or [Input Overview](~/docs/framework/wpf/advanced/input-overview.md).  
-  
+ Even if the element is focusable and valid, the preview event of focus may be processed in a specific tree and focus on that element may not be allowed (for example, in a composite control).
```
Comment (rpetrusha) [inline xml/System.Windows/UIElement.xml]:
```suggestion
 Even if the element is focusable and valid, the `Focus` preview events may be processed in a specific tree, and focus on that element may not be allowed (for example, in a composite control).
```

File: xml/System.Windows/UIElement.xml [hunk 2]
```diff
@@ -1775,10 +1775,13 @@ Image with an Elliptical Clip Region
...
+ In such a case, this method will return `false`.
```
Comment (rpetrusha) [inline xml/System.Windows/UIElement.xml]:
```suggestion
 In such a case, this method returns `false`.
```

File: xml/System.Windows/UIElement.xml [hunk 3]
```diff
@@ -1775,10 +1775,13 @@ Image with an Elliptical Clip Region
...
+ 
+ In general focus is controlled by two different concepts: keyboard focus and logical focus.
+ These two concepts are not always the same.
+ For details, select [Focus summary] (~ / docs / framework / wpf / advanced / focus-overview.md) and [Input summa
```
Comment (rpetrusha) [inline xml/System.Windows/UIElement.xml]:
```suggestion
 For more information, see [Focus overview] (~/docs /framework/wpf/advanced/focus-overview.md) and [Input overview]()~/docs/framework/wpf/advanced/input-overview.md).
```

File: xml/System.Windows/UIElement.xml [hunk 4]
```diff
@@ -1775,10 +1775,13 @@ Image with an Elliptical Clip Region
...
  To be focusable, <xref:System.Windows.UIElement.Focusable%2A> and <xref:System.Windows.UIElement.IsEnabled%2A> must both be `true`.  
   
- Even if an element is focusable and enabled, event handling within a specific tree, (such as for a composite control) might respond to the preview focus events by not allowing focus there, thus this method would return `false`.  
-  
- Focus in general is governed by two separate concepts: keyboard focus and logical focus, which are not always identical.  For more information, see [Focus Overview](~/docs/framework/wpf/advanced/focus-overview.md) or [Input Overview](~/docs/framework/wpf/advanced/input-overview.md).  
-  
+ Even if the element is focusable and valid, the preview event of focus may be processed in a specific tree and focus on that element may not be allowed (for example, in a composite control).
+ In such a case, this method will return `false`.
+ 
+ In general focus is controlled by two different concepts: keyboard focus and logical focus.
```
Comment (rpetrusha) [inline xml/System.Windows/UIElement.xml]:
```suggestion
 In general, focus is controlled by two different concepts: keyboard focus and logical focus.
```

**Comment by mairaw** [comment]
@rpetrusha I've applied your feedback. Can you approve this? Thanks!

**Comment by rpetrusha** [comment]
I've approved it, @mairaw.

**Comment by v-maudel** [comment]
Thanks @rpetrusha and @mairaw !

**Pull request merged**